zerver/views/streams.py

```diff
@@ -314,8 +314,8 @@ def update_stream_backend(
         # Enforce restrictions on creating web-public streams.
         if not user_profile.realm.web_public_streams_enabled():
             raise JsonableError(_("Web public streams are not enabled."))
```

Comment:
This is already checked in `can_create_web_public_streams`. We can keep this if we want to give a different error message or just remove this if we are fine with the same error message.

Reply:
I think this is probably net helpful for the better error message.